provider/datagen/src/transform/cldr/time_zones/convert.rs

```diff
@@ -368,3 +437,63 @@ fn iterate_zone_format_for_time_zone_id(
             )
         })
 }
+
+fn iterate_metazone_period(
+    pair: (
+        TimeZoneBcp47Id,
+        Vec<MetaZoneForPeriod>,
+        LiteMap<String, MetaZoneId>,
+    ),
+) -> impl Iterator<Item = (TimeZoneBcp47Id, String, MetaZoneId)> {
+    let (time_zone_key, periods, meta_zone_id_data) = pair;
+    periods
+        .into_iter()
+        .map(move |period| match &period.uses_meta_zone.from {
+            Some(from) => {
+                match meta_zone_id_data.get(&period.uses_meta_zone.mzone) {
+                    Some(meta_zone_short_id) => (time_zone_key, from.clone(), *meta_zone_short_id),
+                    None => {
+                        // TODO(#1781): Remove this special case once the short id is updated in CLDR
+                        if &period.uses_meta_zone.mzone == "Yukon" {
+                            (
+                                time_zone_key,
+                                from.clone(),
+                                MetaZoneId(tinystr::tinystr!(4, "yuko")),
+                            )
+                        } else {
+                            (
+                                time_zone_key,
+                                from.clone(),
+                                MetaZoneId(tinystr::tinystr!(4, "unkw")),
```

Comment:
I created a string `unkw` as a key if metazone does not have a short id.

Reply:
OK, I think there are two choices here:

1. Define a new string and guarantee that it will not be used in the future by a real metazone by adding that requirement to the spec somewhere
2. Change the provider to be a map to `Option<MetaZoneId>`, and use `None` if we don't have display names for the metazone

Reply:
sg!